app.test.js

```diff
@@ -0,0 +1,36 @@
+import request from 'supertest';
+import app from './app.js';
+import '@babel/polyfill';
+
+const environment = process.env.NODE_ENV || 'development'
+const configuration = require('./knexfile')[environment]
+const database = require('knex')(configuration)
+
+describe('Server', () => {
+
+  beforeEach(async () => {
+    await database.seed.run();
+  });
+
+  describe('init', () => {
+    it('should return a 200 status', async () => {
+      const res = await request(app).get('/')
+      expect(res.status).toBe(200)
+    });
+  });
+
+  describe('GET /api/v1/users/:id', () => {
```

Comment:
Same comment as in `app.js`, this methodology should actually be a POST request.